Cargo.toml

```diff
@@ -13,8 +13,9 @@ edition = "2021"
 rust-version = "1.77"
 default-run = "qdrant"
 
-[lints]
-workspace = true
```

Comment:
solution from https://blog.rust-lang.org/2024/05/06/check-cfg.html#expecting-custom-cfgs

Reply:
Please keep `[lints] workspace = true` as it would disable a bunch of the workspace lints for this crate otherwise. They're listed below in `[workspace.lints.clippy]` section.

Reply:
We can have both by adding this setting to `workspace.lints.rust`. Just pushed the change :)